src/FStringMatchRule.js

```diff
@@ -23,6 +23,21 @@ import { Rule, Result } from 'i18nlint-common';
 // from https://peps.python.org/pep-0498/
 const fstringRegExp = /\{\s*((\}\}|[^}])*?)\s*\}/g;
 
+/**
+ * @private
+ */
+function escapeRegex(re) {
+    return re.
+        replace(/\{/g, "\\{").
+        replace(/\}/g, "\\}").
+        replace(/\[/g, "\\[").
+        replace(/\]/g, "\\]").
+        replace(/\./g, "\\.").
+        replace(/\?/g, "\\?").
+        replace(/\*/g, "\\*").
+        replace(/\+/g, "\\+");
+}
```

Comment:
Shouldn't this also account for `$()|\`?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions#escaping:~:text=function%20escapeRegExp(string)